charts/victoria-metrics-common/templates/_enterprise.tpl

```diff
@@ -27,38 +30,36 @@
 Return license volume mount
 */}}
 {{- define "vm.license.volume" -}}
-{{- $licenseSecretKey := (include "vm.license.secret.key" .) -}}
-{{- $licenseSecretName := (include "vm.license.secret.name" .) -}}
-{{- if and $licenseSecretName $licenseSecretKey -}}
-- name: license-key
-  secret:
-    secretName: {{ $licenseSecretName }}
-{{- end -}}
+  {{- $licenseSecretKey := (include "vm.license.secret.key" .) -}}
+  {{- $licenseSecretName := (include "vm.license.secret.name" .) -}}
+  {{- if and $licenseSecretName $licenseSecretKey -}}
+    {{- $volume := list (dict "name" "license-key" "secret" (dict "secretName" $licenseSecretName)) -}}
+    {{- toYaml $volume -}}
+  {{- end -}}
```

Comment:
Won't this produce exactly the same result as it was before?
It seems to me like plain YAML will be easier to read and understand compared to helm `dict` definition, so I'd keep previous version.

Reply:
restored